charts/open-notificaties/templates/configmap.yaml

```diff
@@ -24,6 +24,15 @@ data:
   {{- end }}
   IS_HTTPS: {{ if .Values.settings.isHttps }}"True"{{ else }}"False"{{ end }}
   RABBITMQ_HOST: {{ .Values.settings.messageBroker.host }}
+  {{- if not eq .Values.settings.autoRetry.maxRetries nil }}
+  NOTIFICATION_DELIVERY_MAX_RETRIES: "{{ .Values.settings.autoRetry.maxRetries }}"
+  {{- end }}
+  {{- if .Values.settings.autoRetry.backoff }}
+  NOTIFICATION_DELIVERY_RETRY_BACKOFF: {{ .Values.settings.autoRetry.backoff }}
+  {{- end }}
```

Comment:
This one is also tricky, in the values yaml this can be a boolean or a number (int or float) - but if it's a boolean, this needs proper casting to a string for environment variables. That should also happen for numbers actually, but ideally I'd like to convert boolean -> `"1"` or `"0", and have numbers as `"3"` for example.